OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -175,8 +175,8 @@ public interface INotifyDeployComplete
 
 	public interface INotifyDeployTriggered
 	{
-		void Deploy(Actor self, bool skipMakeAnim);
-		void Undeploy(Actor self, bool skipMakeAnim);
+		void Deploy(Actor self, string[] deployTypes);
```

Comment:
A `HashSet<string>` would be more appropriate here I think.

Reply:
My initial guess was string[] is faster than HashSet<string> as it is expected that there won't be only a couple of deployTypes in typical use cases. But if you say so.